server/api/sample_sizes.py

```diff
@@ -220,31 +223,40 @@ def serialize_sample_size_options(sample_size_options):
 @api.route("/election/<election_id>/sample-sizes", methods=["GET"])
 @restrict_access([UserType.AUDIT_ADMIN])
 def get_sample_sizes(election: Election):
+    sample_sizes = SampleSizeOptions.query.filter_by(
+        election_id=election.id, round_num=1,
+    ).one_or_none()
+    # If we've never queried sample sizes before for this audit, create a row in
+    # the database to store them.
```

Comment:
Alternatively, could we create the sample sizes record when we create the election (i.e. audit) record? (The current approach seems very reasonable, just curious to hear the reasoning that led to it)

Reply:
Yeah, I considered that. Two reasons I went this direction:
- Next up, we'll be computing sample sizes for later rounds in this same endpoint, so if this endpoint wasn't creating the sample size record for later rounds, it would have to be created elsewhere (e.g. where we finish the previous round). Then we'd have two places that create these records
- It seemed easier to read the code by having all this logic in one place